etc/initdev/init.sh

```diff
@@ -24,7 +24,7 @@ apt-get install -yq --no-install-recommends \
 # installs go
 # we use the beta for now since it will be released in a few weeks
 curl -sSL https://storage.googleapis.com/golang/go${GO_VERSION}.linux-amd64.tar.gz | tar -C /usr/local -xz
-echo 'export PATH=${PATH}:/usr/local/go/bin' >> '/etc/profile'
+echo 'export PATH=${PATH}:/usr/local/go/bin:/home/${1}/go/bin' >> '/etc/profile'
```

Comment:
this is adding a specific users' GOPATH to the global path - adding this somewhere below in `${HOME}/.profile` makes more sense to me